priam/src/main/java/com/netflix/priam/config/PriamConfiguration.java

```diff
@@ -212,11 +212,6 @@ public String getMaxDirectMemory() {
                 (PRIAM_PRE + ".direct.memory.size.") + instanceInfo.getInstanceType(), "50G");
     }
 
-    @Override
-    public String getBackupCronExpression() {
```

Comment:
Why is this removed?

Reply:
It was only referenced in SnapshotBackup which was removed in this cleanup. The modern equivalent is getSnapshotMetaServiceCronExpression() used in SnapshotMetaTask.